cmd/agent/subcommands/secret/command.go

```diff
@@ -84,3 +103,27 @@ func showSecretInfo(config config.Component) error {
 	fmt.Println(string(r))
 	return nil
 }
+
+func secretRefresh(config config.Component) error {
```

Comment:
The implementation of this function is very close to `showSecretInfo`. What do you think about factorising the common logic of these 2 functions?

Reply:
Good idea, done.